app/controllers/lessons_controller.rb

```diff
@@ -0,0 +1,26 @@
+class LessonsController < ApplicationController
+
+  before_action :logged_in_user
+
+  def new
+    @category = Category.find_by_id(params[:category_id])
+    @lesson = Lesson.new
+  end
+
+  def create
+    @category = Category.find(params[:category_id])
+    @lesson = @category.lessons.build(lesson_params)
```

Comment:
@johnedelmapa 
Try building the `lesson` using `current_user`, instead of passing from form user_id